x-pack/plugins/synthetics/public/apps/synthetics/components/monitors_page/management/monitor_list_table/actions.tsx

```diff
@@ -169,20 +126,24 @@ export const Actions = ({ euiTheme, id, name, reloadPage, canEditSynthetics }: P
         <EuiContextMenuPanel size="s" items={menuItems} />
       </EuiPopover>
 
-      {isDeleteModalVisible ? (
-        <EuiConfirmModal
-          title={`${labels.DELETE_MONITOR_LABEL} ${name}`}
-          onCancel={() => setIsDeleteModalVisible(false)}
-          onConfirm={handleConfirmDelete}
-          cancelButtonText={labels.NO_LABEL}
-          confirmButtonText={labels.YES_LABEL}
-          buttonColor="danger"
-          defaultFocusedButton="confirm"
-          isLoading={isDeleting}
-        >
-          <p>{labels.DELETE_DESCRIPTION_LABEL}</p>
-        </EuiConfirmModal>
-      ) : null}
+      {isDeleteModalVisible && (
+        <DeleteMonitor
+          id={id}
+          name={name}
+          reloadPage={reloadPage}
+          isProjectMonitor={isProjectMonitor}
+          setIsDeleteModalVisible={setIsDeleteModalVisible}
+        />
+      )}
     </>
   );
 };
+
+export const PROJECT_MONITOR_DISCLAIMER = i18n.translate(
+  'xpack.synthetics.monitorManagement.monitorList.disclaimer.label',
+  {
+    defaultMessage:
+      'This monitor was added from an external project. Deleting this monitor will not remove it from the project source. ' +
+      'Afterward, you will need to remove it from the project source as well. Otherwise it will be re-added when you use push command from the project source.',
```

Comment:
```suggestion
      'This monitor configuration was pushed from an external project. Deleting it here will not remove it from the project it comes from. ' +
      'To delete it completely or stop it from being added here again in the future, you need to delete it from the external project.',
```
I tried to simplify a bit. Feel free to edit it further or do a mix.